extensions/common/azure/azure-blob-core/src/main/java/org/eclipse/edc/azure/blob/BlobStoreCoreExtensionConfig.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2024 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.azure.blob;
+
+import org.eclipse.edc.runtime.metamodel.annotation.Setting;
+import org.eclipse.edc.runtime.metamodel.annotation.Settings;
+
+@Settings
+public record BlobStoreCoreExtensionConfig(
```

Comment:
this could be called `BlobStorageConfiguration` (I just noticed that we're calling everything BlobStore but the actual product is Blob Storage :'D )

Reply:
You're right! Changed in [here](https://github.com/eclipse-edc/Technology-Azure/pull/298/commits/a20a5bd83a4033c9249b24ca810cbacdc8a248df) :)